server/src/main/java/org/apache/cloudstack/user/UserPasswordResetManager.java

```diff
@@ -55,6 +55,17 @@ public interface UserPasswordResetManager {
             "Use auth in the SMTP server for sending emails for resetting password for ACS users",
             false, ConfigKey.Scope.Global);
 
+    ConfigKey<Boolean> UserPasswordResetSMTPUseStartTLS = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED,
```

Comment:
@weizhouapache do we need to use these settings anywhere ? I don't see any usages!